lock/src/main/java/org/infinispan/lock/impl/lock/ClusteredLockImpl.java

```diff
@@ -59,7 +62,9 @@
    private final EmbeddedClusteredLockManager clusteredLockManager;
    private final FunctionalMap.ReadWriteMap<ClusteredLockKey, ClusteredLockValue> readWriteMap;
    private final Queue<RequestHolder> pendingRequests;
+   private final Queue<UnlockRequestHolder> pendingUnlocks;
```

Comment:
Is this needed? I thought that unlocks were fine during a rehash event?

Reply:
I experienced some issues with unlock too, but not 100% anymore